### PR TITLE
docs(internal): Remove internal team docs after Copybara config change.

### PR DESCRIPTION
**Pull request by joyzhong**
Project import generated by Copybara.

### CODE REVIEW

File: packages/mdc-chips/chip/foundation.ts [hunk 1]
```diff
@@ -71,7 +71,9 @@ export class MDCChipFoundation extends MDCFoundation<MDCChipAdapter> {
     };
   }
 
-  /** Whether a trailing icon click should immediately trigger exit/removal of the chip. */
```
Comment (abhiomkar) [inline packages/mdc-chips/chip/foundation.ts]: Sync to master to revert this change.

File: packages/mdc-chips/chip/foundation.ts [hunk 2]
```diff
@@ -71,7 +71,9 @@ export class MDCChipFoundation extends MDCFoundation<MDCChipAdapter> {
...
+  /**
+   * Whether a trailing icon click should immediately trigger exit/removal of the chip.
+   */
```
Comment (allan-chen) [inline packages/mdc-chips/chip/foundation.ts]: Can we sync to undo this change
